### DISCUSSION TITLE
why isn't bromite blocking youtube ads for me anymore?

### DISCUSSION THREAD

**Discussion by mrawbabylon** [bromite/bromite#1143] [Q&A]
hi! i like to listen to asmr at night and bromite is totally helpful for this because playing in a browser allows me to change tabs or apps and bromite blocks the youtube ads. for some reason, though, youtube ads are playing now and it's a bit irritating... i tried disenabling and re-enabling the ad blocker but it was the same... i apologise in advance if there's a totally simple solution that just didnt occur to me ..

thanks :)

EDIT: should this be posted under issues instead? i'm new to this site lol

**Comment by chromer030** [comment]
Same here.
It's not blocking ads on YouTube anymore.

**Reply by chromer030** [reply]
Please open an issue , as there is a problem about filters i think.

**Reply by lexleephan1712** [reply]
Same here, just the youtube but while it's running in background. It's annoying that now my music has ads in between.

**Reply by mrawbabylon** [reply]
thanks for the response, i'll try to post this on the issues section... hopefully it's not too complicated because i really don't understand things like this lol

**Reply by chromer030** [reply]
I will create an issue with all necessary info and will point it to this discussion.

**Reply by iamthelara** [reply]
Disable/make it automatic: Use secure DNS(dns-over-https), might help.

**Reply by mrawbabylon** [reply]
> Disable/make it automatic: Use secure DNS(dns-over-https), might help.

hi! im not sure what u mean by this. do you mean to write dns://youtube.com or am i misunderstanding?

**Comment by mrawbabylon** [comment]
hey guys... i cant find the issue post for this anymore, but if someone can find it then let me know - im still having this problem and it's terribly frustrating

**Reply by iamthelara** [reply]
Open bromite go to settings>>privacy and security>>Use secure DNS>> Use your current service provider Or disable it.

**Reply by hayaburro** [reply]
@mrawbabylon i test hard lots of .dat filter lists and without ublock origin extension, the only filter list who blocked in-video ads well is [adblockplus filter anti cv](https://easylist-downloads.adblockplus.org/abp-filters-anti-cv.txt) i rarely found some banner ads on youtube but no in-video ads, for me the default easylist, easyprivacy and uAssets have never blocked youtube ads. I recommend [create your own filter list](https://github.com/chromium/chromium/blob/master/components/subresource_filter/FILTER_LIST_GENERATION.md) or [fork this nice proyect](https://github.com/xarantolus/filtrite) to create your own list, i also recommend disable dns over https in bromite and use nebulo + blocking ipv6

**Comment by chromer030** [comment]
@mrawbabylon , Since i changed my filter list to https://github.com/xarantolus/filtrite , i have no problem anymore, while Bromite has by default 8 list of filters , [bromite-extended](https://github.com/xarantolus/filtrite) has 20 list of filters and because of automatic list generation it has very up-to-date filters.

**Reply by csagan5** [reply]
This is false, all filters are affected.

**Reply by chromer030** [reply]
But with default bromite's filters we have video ads on Youtube but with bromite-extended filters no ads anymore

**Reply by csagan5** [reply]
There are ads with both.

**Reply by chromer030** [reply]
So the issue is with Adblocker engine but not filters ?

**Reply by csagan5** [reply]
I do not know what is the issue, it needs investigation.

**Comment by lexleephan1712** [comment]
Ok I'll try that

**Comment by 198tlz** [comment]
Just want to add my own observations here regarding the YouTube ads on my device (currently using the latest v92.0.4515.134 x64 build although the behaviour described below has been happening for a long time now):
- Whenever YouTube is playing an ad at the start of a video, refreshing the page always fixes the issue (ie: once the page reloads, the ad no longer shows and the current YouTube video will play as expected). So this indicates that the ad blocking is partially working in some instances at least
- Using the Bromite Extended filter, the video ad is blocked correctly (ie: ads no longer display at all before any video). But when the ad is blocked by the filter, YouTube will be stuck on an empty/blank video (perhaps because it is trying to play the ad which is now blocked by the filter). When this occurs, I have to refresh the page in order to see the normal YouTube video again

So perhaps, this intermediate state when YouTube is trying to playback the ad is the cause of the issue?

Kind of annoying, especially when trying to play YouTube in the background. Oh well, will stick with another browser for YouTube in the mean time (Firefox/Kiwi plus ublock)
